util/util.go

```diff
@@ -26,7 +26,7 @@ var caretRangesAndPinningRegexp = regexp.MustCompile(`^(\^)?\d(\.\d){2}$`)
 
 // tagRegexp check VERSION of Tags. Tags can only be named with A-Z,a-z,0-9,-
 // ex(latest stable feature-abc)
-var tagRegexp = regexp.MustCompile(`^[a-z][a-z0-9-]+$`)
+var tagRegexp = regexp.MustCompile(`^[a-zA-Z][\w-]+$`)
```

Comment:
This fix is based on https://github.com/screwdriver-cd/data-schema/blob/7554e482917ecb1964bcfcb4c6b656a2f6765c0f/config/regex.js